docs/primitives/index.md

```diff
@@ -8,21 +8,32 @@ order: 1
 section_order: 4
 ---
 
-Primitives alias A-Frame [entities](../core/entity.md) and map HTML attributes to [component](../core/component.md) properties. They are a convenience layer on top of the core API and are meant to:
+Primitives are [entities][entity] that:
 
-- Ease us into the concept of the [entity-component-system](../core/) pattern.
-- Provide a more familiar interface with HTML attributes mapping to only a single value.
-- Pre-compose useful components together with prescribed defaults to create semantic building blocks out-of-the-box.
+- Have a semantic name (e.g., `<a-box>`)
+- Have a preset assemblage of components
+- Have default component property values
+- Map HTML attributes to [component][component] properties
 
-A-Frame ships with a handful of primitives for common use cases such as displaying basic geometric primitives, 3D models, and media assets.
+They are a convenience layer on top of the core API and are meant to:
+
+- Make A-Frame more accessible to beginners and non-technical people
```

Comment:
someone might be technical but want to use primitives. I don't think this is necessarily true even for most folks.